src/test/java/definitions/DemoStepDefs.java

```diff
@@ -65,7 +65,7 @@ public void iClickButton(String btnName) {
     public void confirmationMessageShouldBeDisplayed(String expectedMessage) {
         String actualMessage = getDriver().findElement(By.xpath("//body")).getText();
         System.out.println(getDriver().getCurrentUrl());
-//        assertThat(actualMessage.equals(expectedMessage)).isTrue();
+        assertThat(actualMessage.equals(expectedMessage)).isTrue();
```

Comment:
String actualMessage = getDriver().findElement(By.xpath("//h4")).getText();
//        System.out.println(getDriver().getCurrentUrl()); sthis line should be deleted
    assertThat(actualMessage.equals(expectedMessage)).isTrue();